IFC 4.3/Align_1/UT_Align_1.IFC

```diff
@@ -0,0 +1,103 @@
+/* Generated by TcpMDT - V8.5
+   by APLITOP, S.L. (www.aplitop.com)
+*/
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('UT_Align_1.IFC','2020-11-16T12:56:17',(),(),'IfcOpenShell 0.5.0-dev','IfcOpenShell 0.5.0-dev','');
+FILE_SCHEMA(('IFC4X3_RC2'));
+ENDSEC;
+DATA;
+#1=IFCACTORROLE(.USERDEFINED.,'',$);
+#2=IFCADDRESS(.HOME.,'',$);
+#3=IFCPERSON('Javier','','',$,$,$,(#1),(#2));
+#4=IFCADDRESS(.OFFICE.,'',$);
```

Comment:
An IfcAddress is an abstract class, and only classes inheriting from this should be instantiated such as IfcPostalAddress.  Can you please adjust?  I believe IfcOpenShell has a new audit feature to detect this when reviewing the ifc file if it helps.

Reply:
Hello
First, thank you very much for your comment about audit feature of IfcOpenShell. I will look for it.

I am changing this isssue, and others you told me, but I would like also to include last added attribute to IfcCurveSegment, as @SergejMuhic presented yesterday. 
I see that https://standards.buildingsmart.org/IFC/DEV/IFC4_3/RC2/HTML/link/ifccurvesegment.htm is not updated.
Please, do you know where can I see last changes? In miro board? github? 

Thank you

Reply:
Fixed in 1954276